journeyapps-core-xml/.pnpm-debug.log

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This file needs to be added to `.gitignore`

Reply:
this has been removed.